src/cmd-buildextend-live

```diff
@@ -619,11 +621,30 @@ boot
     with open(miniso_data, 'wb') as f:
         f.truncate(miniso_data_file_size)
 
+    if args.fixture:
+        # Replace or delete anything irrelevant to coreos-installer
+        with open(os.path.join(tmpisoimages, 'efiboot.img'), 'w') as fh:
+            fh.write('efiboot.img\n')
+        with open(os.path.join(tmpisoimagespxe, 'rootfs.img'), 'w') as fh:
+            fh.write('rootfs data\n')
+        with open(os.path.join(tmpisoimagespxe, 'initrd.img'), 'w') as fh:
+            fh.write('initrd data\n')
+        with open(os.path.join(tmpisoimagespxe, 'vmlinuz'), 'w') as fh:
+            fh.write('the kernel\n')
+        with open(os.path.join(tmpisoisolinux, 'isolinux.bin'), 'rb+') as fh:
+            flen = fh.seek(0, 2)
+            fh.truncate(0)
+            fh.truncate(flen)
+        for f in ensure_glob(os.path.join(tmpisoisolinux, '*.c32')):
+            os.unlink(f)
+        for f in ensure_glob(os.path.join(tmpisoisolinux, '*.msg')):
+            os.unlink(f)
+
     run_verbose(genisoargs_final)
 
     # Add MBR, and GPT with ESP, for x86_64 BIOS/UEFI boot when ISO is
     # copied to a USB stick
-    if basearch == "x86_64":
+    if basearch == "x86_64" and not args.fixture:
```

Comment:
Hmm, maybe better to just leave those on? How much data does it add? The other bits above were just unlinking files but `isohybrid` is lower-level so would be good to have it stay faithful in that respect.

Reply:
Yeah, I'd prefer to leave it in.  `isohybrid` wants a valid `isolinux.bin`, and that's 38 K and doesn't compress well.  I could look into what validity checks it's doing.

Reply:
It's just a magic number check, which I've now circumvented.  `isohybrid` adds 1076 bytes (18%) to the compressed ISO.